lifemonitor/utils.py

```diff
@@ -151,6 +151,13 @@ def _download_from_remote(url, output_stream, authorization=None):
                 output_stream.write(chunk)
 
 
+def check_resource_exists(url):
+    r = requests.head(url, verify=False)
+    result = r.status_code != 404
```

Comment:
I think the check should be `r.status_code == 200`. You cannot state the resource exists if you get an error other than 404 (especially if you get a 410).

Reply:
fixed in https://github.com/crs4/life_monitor/pull/146/commits/49ed591e27f06ef81cf6943168de80b3c22da997